src/containers/audio-rewards-page/components/modals/RewardsModals.tsx

```diff
@@ -7,12 +7,20 @@ import TransferAudioMobileDrawer from './TransferAudioMobileDrawer'
 import TrendingRewardsModal from './TrendingRewards'
 import VerifiedUpload from './VerifiedUpload'
 
+const HCaptchaModal = React.lazy(() => import('./HCaptchaModal'))
```

Comment:
this should be fine, but you may want to consider lazyWithPreload if the user hits the parent component here to start the fetching of the modal/hcaptcha code and then have a separate condition to actually do the modal display logic